packages/E2ETest/windows/ReactUWPTestApp/ReactUWPTestApp.csproj

```diff
@@ -108,6 +108,8 @@
     <None Include="ReactUWPTestApp_TemporaryKey.pfx" />
   </ItemGroup>
   <ItemGroup>
+    <Content Include="Assets\TreeDump\ControlStyleRegularBorder.txt" />
+    <Content Include="Assets\TreeDump\ControlStyleRoundBorder.txt" />
     <Content Include="Assets\TreeDump\ImageWithBorder.txt" />
     <Content Include="Assets\TreeDump\ImageWithoutBorder-Subsequent.txt" />
     <Content Include="Assets\TreeDump\ImageWithoutBorder.txt" />
```

Comment:
<Content Include="Assets\TreeDump\*.txt" />
We don't need to update csproj everytime

Reply:
Done, thanks!

---
In reply to: [359599465](https://github.com/microsoft/react-native-windows/pull/3793#discussion_r359599465) [](ancestors = 359599465)